CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## TBD
+
+* Increase kotlin dependency version to 1.3.72
```

Comment:
I'm not sure whether this needs a changelog entry though as it doesn't affect our users?

Reply:
There shouldn't be any effect on the API bugsnag uses, but the Kotlin compiler won't produce identical bytecode for 1.3.61 to 1.3.72, so I thought it'd be sensible to document as there is the small possibility it could introduce change. Adding a changelog entry is also consistent with what was done when upgrading previously